purchases/src/main/java/com/revenuecat/purchases/interfaces/ProductChangeListener.java

```diff
@@ -0,0 +1,25 @@
+//  Purchases
+//
+//  Copyright © 2019 RevenueCat, Inc. All rights reserved.
+//
+
+package com.revenuecat.purchases.interfaces;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.android.billingclient.api.Purchase;
+import com.revenuecat.purchases.PurchaserInfo;
+
+/**
+ * Interface to be implemented when upgrading or downgrading a purchase.
+ */
+public interface ProductChangeListener extends PurchaseErrorListener {
```

Comment:
there's a new public method, but it's backward-compatible, so the milestone should be 3.6.0, right?